test/passing/tests/comments_in_record.ml.ref

```diff
@@ -48,7 +48,9 @@ type t =
   | Struct of
       { name: string
       ; elts: t vector (* possibly cyclic, name unique *)
-            [@compare.ignore] [@equal.ignore] [@sexp_drop_if fun _ -> true]
+            [@compare.ignore]
+            [@equal.ignore]
+            [@sexp_drop_if fun _ -> true]
```

Comment:
This is a new behavior for attributes. I think the wrapping should be re-implemented (only on `@` attributes).

Reply:
reverted